Update readme files (#2)
DevExpress-Examples/getting-started-with-dropdownbutton: angular/README.md

# DevExtreme Angular DropDownButton - Getting Started

> This repository stores the code examples of the Angular DropDownButton component for the [Getting Started with DropDownButton](https://js.devexpress.com/Documentation/Guide/UI_Components/DropDownButton/Getting_Started_with_DropDownButton/) tutorial. The DropDownButton combines the functionality of a button and a drop-down menu. This tutorial describes how to configure a DropDownButton that logs user clicks in the browser console.

Reviewer: ```suggestion
This repository stores code examples of the Angular DropDownButton component for the [Getting Started with DropDownButton](https://js.devexpress.com/Documentation/Guide/UI_Components/DropDownButton/Getting_Started_with_DropDownButton/) tutorial. The DropDownButton combines the functionality of a button and a drop-down menu. This tutorial describes how to configure a DropDownButton that logs user clicks in the browser console.
```

## Running unit tests

Run `ng test` to execute unit tests using [Karma](https://karma-runner.github.io).

## Running end-to-end tests

Run `ng e2e` to execute end-to-end tests using [Protractor](http://www.protractortest.org/).

## Further help

> To get more help on the Angular CLI, use `ng help` or review the [Angular CLI Overview and Command Reference](https://angular.io/cli) page.

Reviewer: ```suggestion
For help with Angular CLI, use `ng help` or review the [Angular CLI Overview and Command Reference](https://angular.io/cli) page.
```
